### ISSUE TITLE
Smaller way to store the card on the dashboard

### ISSUE THREAD

**Issue by Artropos** [pathofleastresistor/polr-ytube-media-card#10]
Could you make a way to make the card smaller?
Like now its this as default
![image](https://github.com/pathofleastresistor/polr-ytube-media-card/assets/149838644/63af10ae-e7d1-4528-aa36-ec03af3ed729)

But is there a way you can make it like this
![image](https://github.com/pathofleastresistor/polr-ytube-media-card/assets/149838644/7755650f-efaf-404a-85ed-0546208b13cf)
And that when you click on Playing (Better to call it search if you change it) it will extend and open the menu from above

**Comment by pathofleastresistor** [comment]
Hmm I'm open to considering some alternative ways to consider a more compact card. Can I ask how you lay out the cards around this card? I'm just wondering how your layout would be impacted if the height changed dynamically.

**Comment by Thorsten1982** [comment]
Yes, I would like that too.  I think the playback devices could be removed, as they can already be selected with the button at the top right.  Maybe you could add buttons for Library, Uploaded etc.  Or it wouldn't be bad if you could choose what you want to see in the configuration.  So for example uploaded songs yes/no...

**Comment by Artropos** [comment]
@pathofleastresistor Atm its too big to add it too a nice dashboard with the playing etc always open, if its a bit tinyer and you can fold open the playing/for you part it would be much better

![image](https://github.com/pathofleastresistor/polr-ytube-media-card/assets/149838644/ee613dca-b45d-4703-aed2-c3fbede37356)

Like this when your just playing

And when you click on the playing it extends to this

![image](https://github.com/pathofleastresistor/polr-ytube-media-card/assets/149838644/01875071-d6e1-4190-9329-23b932b3976a)

I would also recommend to put the search bar above playing/for you.
And when you type it auto opens the playing with your search

For anything else would say i love to use this, because its much better then the other players i found around
